src/go/rpk/pkg/cli/cmd/cluster/partitions/cancel.go

```diff
@@ -0,0 +1,84 @@
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewMovementCancelCommand(fs afero.Fs) *cobra.Command {
+	var (
+		node      int
+		noConfirm bool
+	)
+	cmd := &cobra.Command{
+		Use:   "movement-cancel",
+		Short: "Cancel ongoing partitions reconfigurations",
+		Long: `Cancel ongoing partitions reconfigurations happening in the cluster
+
+By default, this command cancels all the partitions reconfiguration in the
+cluster, to ensure that you do not accidentally cancel all movements, this
+command prompts for confirmation before issuing the cancellation request to the
+cluster, you can use "--no-confirm" to disable the confirmation prompt:
+
+    rpk cluster partitions movement-cancel --no-confirm
+
+If --node is set, this command will only stop the partitions happening in the
+specified node and it would not prompt for confirmation:
```

Comment:
Why not prompt?

Also, `will not`, although I think this should still prompt.

Reply:
Just added the prompt for `--node` option.